src/overview/helpers.ts

```diff
@@ -19,3 +20,49 @@ export function safeDecodeURI(data: string): string {
     return data
   };
 };
+
+export function prettyQuestion(data: PlanXExportData["question"]): string {
+  if (data.includes("?") || data.includes("File") || (!data.includes("_") && data.includes(" "))) {
+    return safeDecodeURI(data);
+  } else {
+    return safeDecodeURI(prettyTitle(data));
+  };
+};
+
+export function prettyResponse(data: PlanXExportData["responses"]): any {
```

Comment:
this new helper function borrows a lot from your previous DataItem, but since we're omitting some of the more complex data points from the list now (eg the gov pay response object), is a bit more straightforward